components/01-globals/typography/label/_label.scss

```diff
@@ -0,0 +1,13 @@
+.label {
+    display: block;
+    margin-bottom: 8px;
```

Comment:
Each of these parameters (maybe excluding `display`) should be a variable with `!default` and then you will not have to overwrite this component in child components.